cluster/examples/kubernetes/ceph/create-external-cluster-resources.sh

```diff
@@ -57,15 +62,25 @@ function createCephCSIKeyringCephFSProvisioner() {
   echo "export CSI_CEPHFS_PROVISIONER_SECRET=$cephCSIKeyringCephFSProvisionerKey"
 }
 
+function getMonIP () {
+  monip=$(ceph mon stat | awk '{ print $5 }' | awk -F ':' '{ print $2 }')
+  echo "export ROOK_EXTERNAL_CEPH_MON_DATA=a=$monip:6789"
+}
 
 ########
 # MAIN #
 ########
 checkEnv
+getFSID
 createCheckerKey
 createCephCSIKeyringRBDNode
 createCephCSIKeyringRBDProvisioner
 createCephCSIKeyringCephFSNode
 createCephCSIKeyringCephFSProvisioner
+getMonIP
+
+echo "export NAMESPACE=rook-ceph"
```

Comment:
Why is the script just `echo`ing these exports?

Reply:
Ömmmm ... the create-external-cluster-resources.sh should put out the exports for the import-external-cluster.sh. And the import-external-cluster.sh need this 3 exports too. I have not touch the import-external-cluster.sh script.

Reply:
PS: i have change this anyway. My change make no export anymore. The Problem from export is any User (Gast, low User) that can execute printenv can read the Token. My change echo the Variable Par only. then i can make:
kubectl -n rook-ceph exec -t ceph-toolbox --  create-external-cluster-resources.sh >cluster-var.conf. then i have a file that can insert with:
. cluster-var.conf
in the import-external-cluster.sh. This file i can make root only and 600 to hide it for unauthorized User.